groups/infrastructure/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
```

Comment:
The `required_version` and `version` declarations should specify a suitable lowerr and upper bound, as per our best practices.

Also consider using updated provider versions, if possible.

E.g.:
```
terraform {
  required_version = "> 1.3, < 2.0"
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "> 5.0, < 6.0"
    }
    vault = {
      source  = "hashicorp/vault"
      version = "> 4.0, < 5.0"
    }
  }
}
```

Reply:
This is fixed now. Commit is pushed.